src/main/kotlin/xyz/crossward/controller/GroupController.kt

```diff
@@ -0,0 +1,50 @@
+package xyz.crossward.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.*
+import xyz.crossward.entities.Group
+import xyz.crossward.service.GroupService
+import java.net.URLDecoder
+
+@RestController
+@RequestMapping("/api")
```

Comment:
You can make this `/api/groups` so you don't have to repeat it for each endpoint